src/modules/formBuilder/components/formTree/FormTreeLabel.tsx

```diff
@@ -67,6 +69,19 @@ const FormTreeLabel: React.FC<FormTreeLabelProps> = ({
 
   const labelProps = getLabelProps();
 
+  const upRef = useRef<HTMLButtonElement | null>(null);
+  const downRef = useRef<HTMLButtonElement | null>(null);
+
+  // When the user reorders form items, we want to maintain focus on the up/down button in question. But, since some
+  // form reorder actions use rhf's `reset` on the whole structure, this refocus lives here in this useEffect.
+  useEffect(() => {
+    if (focusedTreeButton === `${itemId}-up`) {
+      upRef.current?.focus();
+    } else if (focusedTreeButton === `${itemId}-down`) {
+      downRef.current?.focus();
+    }
+  }, [upRef, downRef, focusedTreeButton]);
```

Comment:
GitHub actions is pointing out missing dependencies. These show up as warnings with `yarn lint` too. I have my editor configured to give me squiggly lines here based on our `.eslintrc.js`, which is helpful (And can auto-correct it).  If you have a reason for skipping something in the dependency array, can you add an eslint-ignore with an explanation of why?

Reply:
Thanks 😵 yes I noticed my ide stopped warning me about eslint issues and it's been on my list to figure out why, thanks for the push.

Could we add this to our pre-commit hook also, using the ` --max-warnings=0` flag? https://stackoverflow.com/questions/51158815/running-eslint-in-precommit-does-not-stop-on-warnings

Reply:
see separate PR: https://github.com/greenriver/hmis-frontend/pull/833